src/Plutus/Certification/Cicero.hs

```diff
@@ -105,6 +107,22 @@ ciceroServerCaps backend CiceroCaps {..} = ServerCaps {..}
           runs <- lift . runClientOrDie clientCaps eb $ ciceroClient.run.getAll True [rid'] (Just offset) (Just limit)
           count <- yieldMany runs .| execStateC 0 (status eb)
           when (count == limit) $ go (offset + limit)
+    abortRuns mods rid = go 0
+      where
+        eb = modifyEventBackend mods backend
+        rid' = Cicero.Fact.FactID $ rid.uuid
+        limit = 10
+        go offset = do
+          runs <- lift . runClientOrDie clientCaps eb $ ciceroClient.run.getAll True [rid'] (Just offset) (Just limit)
+          let count = List.genericLength runs
+          yieldMany runs
+            .| filterC (isNothing . (.finishedAt))
```

Comment:
Note that in principle anyone could set up arbitrary cicero jobs downstream of our facts. But I don't think this will be an issue and I'd like to get off of cicero anyway.